Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11
```

Comment:
Consider using a slim version of the Python image, such as `FROM python:3.10.8-slim`, to reduce the image size and adhere to best practices.